crmsh/bootstrap.py

```diff
@@ -186,6 +190,20 @@ def init_sbd_manager(self):
         from .sbd import SBDManager
         self.sbd_manager = SBDManager(self)
 
+    def load_profiles(self):
+        """
+        Load profiles data for different cloud provider
+        """
+        if not os.path.exists(PROFILES_FILE):
+            return
+        env_name = utils.detect_cloud()
+        if not env_name:
+            env_name = USER_CUSTOMIZED
+        with open(PROFILES_FILE) as f:
+            data = yaml.load(f, Loader=yaml.SafeLoader)
+        if env_name in data:
```

Comment:
Since we don't have profiles of GCP and AWS, here should check if the key in the dict

Reply:
make sense